environment/src/main/java/com/sequenceiq/environment/environment/domain/Environment.java

```diff
@@ -85,6 +87,10 @@ public class Environment implements AuthResource, AccountAwareResource {
 
     private String freeIpaImageId;
 
+    @ElementCollection(fetch = FetchType.EAGER)
+    @Column(name = "recipe")
+    private Set<String> freeipaRecipes;
```

Comment:
I'm not sure if it's a good idea to always fetch these for the env entity. couldn't this be loosely coupled? Like fetch freeipa recipes separately when required